tests/ignite/contrib/handlers/test_base_logger.py

```diff
@@ -154,29 +153,29 @@ def update_fn(engine, batch):
         mock_log_handler.assert_called_with(trainer, logger, event)
         assert mock_log_handler.call_count == n_calls
 
-    n_iterations = 10
-    cpe1 = CustomPeriodicEvent(n_iterations=n_iterations)
-    n = len(data) * n_epochs / n_iterations
-    nf = math.floor(n)
-    ns = nf + 1 if nf < n else nf
-    _test(cpe1.Events.ITERATIONS_10_STARTED, ns, cpe1)
-    _test(cpe1.Events.ITERATIONS_10_COMPLETED, nf, cpe1)
-
-    n_iterations = 15
-    cpe2 = CustomPeriodicEvent(n_iterations=n_iterations)
-    n = len(data) * n_epochs / n_iterations
-    nf = math.floor(n)
-    ns = nf + 1 if nf < n else nf
-    _test(cpe2.Events.ITERATIONS_15_STARTED, ns, cpe2)
-    _test(cpe2.Events.ITERATIONS_15_COMPLETED, nf, cpe2)
-
-    n_custom_epochs = 2
-    cpe3 = CustomPeriodicEvent(n_epochs=n_custom_epochs)
-    n = n_epochs / n_custom_epochs
-    nf = math.floor(n)
-    ns = nf + 1 if nf < n else nf
-    _test(cpe3.Events.EPOCHS_2_STARTED, ns, cpe3)
-    _test(cpe3.Events.EPOCHS_2_COMPLETED, nf, cpe3)
+        n_iterations = 10
```

Comment:
I think all those indents are incorrect. Function `_test` is called in various confs.

Reply:
They were accidentally indented, but it should be fixed now